hil-test/tests/gpio.rs

```diff
@@ -27,12 +27,15 @@ static INPUT_PIN: Mutex<RefCell<Option<Input<'static, TestGpio1>>>> =
     Mutex::new(RefCell::new(None));
 
 cfg_if::cfg_if! {
-    if #[cfg(not(any(esp32s2, esp32s3)))] {
+    if #[cfg(not(any(esp32, esp32s2, esp32s3)))] {
```

Comment:
Nitpick (feel free to ignore it), maybe we could check esp32, then s2/s3 and in the else clause use pins 2-3 so we avoid the `not(any...)` check

Reply:
Updated, thanks